src/types/colonyActions.ts

```diff
@@ -121,6 +124,8 @@ export interface FormattedAction {
   transactionHash: string;
   createdAt: Date;
   commentCount: number;
+  role: ColonyRole;
+  setTo: string;
```

Comment:
why string? 🤔

Reply:
See [this](https://github.com/JoinColony/colonyDapp/pull/2355#discussion_r570970278) comment